### PR TITLE
Fix errors during ADU update

### PR DESCRIPTION
**Pull request by willdurand**
Fixes https://github.com/mozilla/addons-server/issues/14650

### CODE REVIEW

File: src/olympia/stats/tests/test_utils.py [hunk 1]
```diff
@@ -236,3 +236,15 @@ def test_returned_results(self, bigquery_client_mock):
 
         returned_results = get_addons_and_average_daily_users_from_bigquery()
         assert returned_results == [(1, 123), (2, 456)]
+
+    @mock.patch('google.cloud.bigquery.Client')
+    def test_skips_null_values(self, bigquery_client_mock):
+        results = [
+            self.create_bigquery_row({'addon_id': 1, 'count': 123}),
+            self.create_bigquery_row({'addon_id': 2, 'count': None}),
+        ]
```
Comment (eviljeff) [inline src/olympia/stats/tests/test_utils.py:245]: isn't this testing the wrong thing? It's `None` values in the database rather the bigquery that are failing.

Comment (willdurand) [reply src/olympia/stats/tests/test_utils.py:245]: BigQuery can return `count=None` and the DB can return `guid=None`

Comment (willdurand) [reply src/olympia/stats/tests/test_utils.py:245]: We should probably have a test case for the DB change but simply adding an add-on without a GUID does not seem to work 🤷

Comment (eviljeff) [reply src/olympia/stats/tests/test_utils.py:245]: > 
> 
> BigQuery can return `count=None` and the DB can return `guid=None`

right, sure, but the change above is only related to the database query, but this test only (appears to) mock bigquery response.

Comment (eviljeff) [reply src/olympia/stats/tests/test_utils.py:245]: > 
> 
> We should probably have a test case for the DB change but simply adding an add-on without a GUID does not seem to work 🤷

what do you mean doesn't work?  You can't `.update(guid=None)`?

Comment (willdurand) [reply src/olympia/stats/tests/test_utils.py:245]: Yeah, that works (or `addon_factory(guid=None)`) but for some reason, we don't get an error raised by the subtask in this cron task..

Comment (willdurand) [reply src/olympia/stats/tests/test_utils.py:245]: I added another test case.

**Pull request merged**